server.js

```diff
@@ -24,9 +23,7 @@ const notes = {
 };
 
 
-const getNextId = () => _.parseInt(
-  _.maxBy(Object.keys(notes), _.parseInt)
-) + 1;
+const getNextId = () => Number.parseInt(Math.max(...Object.keys(notes).map(Number.parseInt)), 10) + 1;
```

Comment:
not related to the removal of `lodash` but what are your thoughts about adding a comment for what `getNextId` is doing and changing the method name to `createNextId`.

In the context of an interview, seeing a formula like this can be intimidating in my opinion.  However I have found that when I say this is simply a way for us to create a new ID, it takes away a lot of the pressure.  Also seeing the `get` in the method name implies that the id already exists rather than needs to be created.  
```suggestion
// Creates a new ID since we are not actually connecting to a database 
const createNextId = () => Number.parseInt(Math.max(...Object.keys(notes).map(Number.parseInt)), 10) + 1;
```

Reply:
Definitely appreciate the suggestion! In this particular scenario I'd argue we should stick to keeping the scope of this as removing lodash.

I've noted that there are not any comments in the repo as it stands, and I wouldn't want to deviate from the existing conventions unless we explicitly have agreed that it makes sense!